content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst

```diff
@@ -1,18 +1,144 @@
+address, where a company's items are stored.
+
+Configure :doc:`routes <inventory_management/use_routes>` in a warehouse to control how products
+move to customers, from vendors, within the warehouse, or :doc:`between warehouses
+<inventory_management/resupply_warehouses>`.
+
+Locations
+=========
+
+:doc:`Locations <inventory_management/use_locations>` refer to specific areas within a warehouse,
+such as shelves, floors, or aisles. These are sub-divisions within a warehouse, and are unique to
+that warehouse. Users can create and manage numerous locations within a single warehouse to organize
+inventory more precisely.
+
+.. seealso::
+   - :doc:`inventory_management/use_locations`
+   - :doc:`inventory_management/count_products`
+   - :doc:`inventory_management/cycle_counts`
+   - :doc:`inventory_management/scrap_inventory`
+
+.. _inventory/warehouses_storage/location-type:
+
+Location types
+--------------
+
+*Location types* in Odoo help categorize and manage where products are, and what actions need to be
+taken with them. By default, on the :menuselection:`Inventory app --> Configuration --> Locations`
+page, only internal locations are displayed.
+
+To view the seven location types in Odoo, select any location, and in the :guilabel:`Location Type`
+field, there are:
+
+- :guilabel:`Vendor Location`: defines an area where products purchased from vendors originate.
+  Items here are **not** in stock.
+
+- :guilabel:`View`: used to organize and structure the warehouse hierarchy. For example, the view
+  location `WH` (short for warehouse) groups all internal locations, such as `Stock`, receiving
+  docks, quality checkpoints, and packing areas to show they all belong to the same warehouse.
+
+  .. important::
+     View locations should **not** contain products, but it is possible to move them there.
+
+- :guilabel:`Internal Location`: storage locations within the warehouse. Items stored in these
+  locations are accounted for in :doc:`inventory valuation
+  <inventory_valuation/using_inventory_valuation>`.
+
+- :guilabel:`Customer Location`: where sold products are tracked; items here are no longer in stock.
+
+- :guilabel:`Inventory Loss`: counterpart location to consume missing items or create stock,
+  accounting for discrepancies.
+
+  In Odoo, examples of inventory loss locations are *Inventory Adjustment*, used to account for
+  discrepancies during an inventory count, and *Scrap*, which is where damaged goods are sent to
+  account for inventory losses.
+
+   .. example::
+      `Virtual Locations/Inventory Adjustment` is a location with the :guilabel:`Inventory Loss`
+      type. The database shows `65` units in `WH/Stock`, but an inventory check reveals `60`. To
+      correct the quantity, five units are moved from `WH/Stock` to `Virtual Locations/Inventory
+      Adjustment`.
+
+      .. image:: inventory_management/inventory_management/inventory-loss.png
```

Comment:
this screenshot is a bit hard to read 
consider making it larger and zooming in if you can.

Reply:
done!